follow/src/main/resources/application.yml

```
@@ -18,15 +18,28 @@ spring:
+      host: localhost
+      port: 6379
+      password: ${REDIS_PASSWORD}
   kafka:
     bootstrap-servers: localhost:9092
     producer:
       key-serializer: org.apache.kafka.common.serialization.StringSerializer
       value-serializer: org.springframework.kafka.support.serializer.JsonSerializer
     consumer:
       key-deserializer: org.apache.kafka.common.serialization.StringDeserializer
-      value-deserializer: org.springframework.kafka.support.serializer.JsonDeserializer
+#      value-deserializer: org.springframework.kafka.support.serializer.JsonDeserializer
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)